test/utils/getCalendarMonthWeeks_spec.js

```diff
@@ -38,6 +38,32 @@ describe('getCalendarMonthWeeks', () => {
     });
   });
 
+  describe('padding when enableOutsideDays is false', () => {
+    let weeksWithPadding;
+
+    before(() => {
```

Comment:
this should use `beforeEach` - `before` means "before all" so it only runs once.